spinalcordtoolbox/reports/qc.py

```diff
@@ -661,6 +666,12 @@ def qcslice_layout(x): return x.mosaic()
         qcslice_type = qcslice.Axial([Image(fname_in1), Image(fname_seg)])
         qcslice_operations = [QcImage.template]
         def qcslice_layout(x): return x.mosaic()
+    # Axial orientation, switch between gif image (before and after motion correction) and grid overlay
+    elif process in ['sct_dmri_moco', 'sct_fmri_moco']:
+        plane = 'Axial'
+        qcslice_type = qcslice.Axial([Image(fname_in1), Image(fname_in2), Image(fname_seg)])
+        qcslice_operations = [QcImage.grid]  # grid will be added in future PR
+        def qcslice_layout(x): return x.mosaic_through_time()  # mosaic_through_time will be added in future PR
```

Comment:
I'm hesitant about approving/merging this. If this is merged, then until the other work is finished, our release process gets more complicated. We would have to choose between:

* Creating a release with a half-completed feature. (Currently, I think calling `sct_qc -p sct_dmri_moco` would fail without much indication to the user as to why.)
* Delaying the release until the other parts are finished.

Reply:
If it's okay with @jcohenadad maybe I could do one branch for the whole issue #3242 and make sure to keep it up-to-date with the master.

Reply:
> I could do one branch for the whole issue #3242 and make sure to keep it up-to-date with the master.

I think this sounds good, although there might be a good middle ground, too. (e.g. 2-3 PRs instead of 1 or 8.) 

Breaking up features like this can be tricky... There is a trade-off between "splitting up work so that PRs are easier to review" and "grouping work, so that it can be reviewed/tested together." There are [lots of factors](https://opensource.com/article/18/6/anatomy-perfect-pull-request) to think about. :sweat_smile: